cotainr/cli.py

```diff
@@ -51,7 +51,7 @@
 class CotainrSubcommand(ABC):
     """Abstract base class for `CotainrCLI` subcommands."""
 
-    @classmethod
+    @classmethod  # noqa: B027
```

Comment:
I guess I was a little too fast here. It seems that this suggested change has made quite a lot of tests fail since we actually don't expect `CotainrSubcommand` subclasses to implement the `add_arguments()` method if they take no arguments. I think we should stick with your proposal of just adding a `# noqa: B027`.